modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/layout/index.css

```
@@ -167,7 +170,7 @@
 	font-weight: 700;
 }
 
-@media screen and (max-width: 1625px) {
+@media creen and (max-width: 1625px) {
```

Please, believe this change was made in the wrong way. 
```suggestion
@media screen and (max-width: 1625px) {
```